### PR TITLE
Initial wof

### LINKED ISSUE #26: Create WOF spinner subsystem and commands
This has already been planned out. See @AkashaSnow or @bobtheragelot for the plan.

See Structure.md for more info about the Wheel of Fortune Spinner.

### PR DESCRIPTION
**Pull request by Dustin-Klein**
resolves #26 
spinToColor and spinToCount functionality.
changes made by Ryan

### CODE REVIEW

File: src/main/java/frc/robot/commands/SpinToColor.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    return findColor.equals(colors[findIndex]);
+  }
+
+  @Override
+  public void end(boolean interrupted) {
+    wallOfFlesh.runWOFSpinner(0);
+  }
+}
+
+
```
Comment (Dustin-Klein) [inline src/main/java/frc/robot/commands/SpinToColor.java]: blank lines at the end of the file

File: src/main/java/frc/robot/commands/SpinToColor.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  private WallOfFlesh wallOfFlesh;
+  private final int colorOffset = 1;
+  private Color[] colors;
+  private int findIndex;
+
+  private Color findColor;
+
+  /**
+   * Constructor.
+   * @param color color to spin to
```
Comment (Dustin-Klein) [inline src/main/java/frc/robot/commands/SpinToColor.java:24]: bad spacing.  Did u ctrl + alt + L

File: src/main/java/frc/robot/commands/SpinToCount.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    return false;
+  }
+
+  @Override
+  public void end(boolean interrupted) {
+    wallOfFlesh.runWOFSpinner(0);
+  }
+}
+
+
```
Comment (Dustin-Klein) [inline src/main/java/frc/robot/commands/SpinToCount.java]: blank lines bad

File: src/main/java/frc/robot/subsystems/ColorMatcher.java [hunk 4]
```diff
@@ -0,0 +1,82 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.ColorMatch;
+import com.revrobotics.ColorMatchResult;
+import com.revrobotics.ColorSensorV3;
+import edu.wpi.first.wpilibj.I2C;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj.util.Color;
+
+
```
Comment (Dustin-Klein) [inline src/main/java/frc/robot/subsystems/ColorMatcher.java]: bad spacing

File: src/main/java/frc/robot/subsystems/WallOfFlesh.java [hunk 5]
```diff
@@ -7,14 +7,75 @@
...
+   * Spins the WOF spinner distance in meters.
+   *
+   * @param distance distance in meters
+   */
+  public void spinDistance(double distance) {
+    int goalEncoderTick = (int) ((distance / (2 * Math.PI * WallOfFleshConstants.spinnerRadius))
+        * 4096);
+    WOFMotor.getSelectedSensorPosition();
+    WOFMotor.set(ControlMode.Position, WOFMotor.getSelectedSensorPosition() + goalEncoderTick);
+  }
```
Comment (Dustin-Klein) [inline src/main/java/frc/robot/subsystems/WallOfFlesh.java:79]: Is meters really the unit we want to use here?

File: src/main/java/frc/robot/subsystems/WallOfFlesh.java [hunk 6]
```diff
@@ -7,14 +7,75 @@
...
+   * Makes WOF a singleton.
+   *
+   * @return WOF instance.
+   */
+  public static WallOfFlesh getInstance() {
+    if (instance == null) {
+      instance = new WallOfFlesh();
+    }
+    return instance;
   }
```
Comment (Dustin-Klein) [inline src/main/java/frc/robot/subsystems/WallOfFlesh.java]: Please move this up to the first method under the constructor

File: src/main/java/frc/robot/subsystems/WallOfFlesh.java (hunk 5 above)
Comment (SHS-Programmers) [reply src/main/java/frc/robot/subsystems/WallOfFlesh.java:79]: YES

**Pull request merged**